samples/Kconfig

```diff
@@ -16,14 +16,58 @@ config NCS_SAMPLES_DEFAULTS
 	help
 	  Use the default configuration for NCS samples.
 
-config NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE
-	bool "Enable empty_app_core as a child image"
-	default n
+if SOC_SERIES_NRF53X
+
+config NCS_CHILD_IMAGE_SAMPLE_EMPTY_APP_CORE
+	bool "Add empty_app_core as a child image"
 	depends on SOC_NRF5340_CPUNET
 	select PARTITION_MANAGER_ENABLED
 	help
-	  Add the empty_app_core as a child image for the given sample.
-	  Used by samples that only require SOC_NRF5340_CPUNET.
+	  Add empty_app_core as a child image (application core).
+	  Used by samples that run on the network core only.
+
+config NCS_AUTO_INCLUDE_NET_CORE_CHILD_IMAGE
+	bool "Automatically add a child image to the network core when relevant"
```

Comment:
This title is still not clear to an outside user.

What is auto, and what child images will be included ?

Now that Kconfig has been cleaned up to only cover the `RPMsg` child images, let's pass that info to the user.

```suggestion
	bool "Include RPMsg sample on netcore"
```